docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,98 @@
+Policy for Unofficial and Outdated Projects
+===========================================
+
+This policy describes a process where we take actions against unmaintained and unofficial forks of project documentation.
```

Comment:
Add links to the sections of this page that explains these concepts. Also, it could go one step further and use the glossary as well.

```suggestion
This policy describes a process where we take actions against unmaintained_ and unofficial_ forks of project documentation.

.. _unmaintained: #unmaintained-projects
.. _unofficial: #unofficial-projects
```

Reply:
Good change, it's important that we are using words signalling that they have a definition, and the link helps accomplish this :+1: 

But I think it's problematic if the reader has to leave this article to understand the meaning of its 2 most central terms, so I don't think they should exist in the Glossary :) However, imagining a future where these definitions work with sphinx-hoverxref or some other inclusion method, that'd be cool for legal and academic writing :+1:

Reply:
> imagining a future where these definitions work with sphinx-hoverxref or some other inclusion method, that'd be cool for legal and academic writing

They already work, that's the point. Not having to jump into another page 😄 


![Screenshot_2022-11-16_12-12-23](https://user-images.githubusercontent.com/244656/202165746-043352d0-933f-4d10-a755-ba739a2c1d1d.png)

However, my proposal here is link to the section inside the same page. We have titles for "Unmaintained projects" and "Unofficial projects" in this page